Appy.GitDb.Core/Model/MergeInfo.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+
+namespace Appy.GitDb.Core.Model
+{    
+    /// <summary>
+    /// The status of what happened as a result of a merge.
+    /// </summary>
+    public enum MergeStatus
```

Comment:
Maybe rename to MergeResult. Result indicates it's final whereas status may hint that it can still change